solarforecastarbiter/io/tests/test_io_utils.py

```diff
@@ -407,6 +407,28 @@ def f(other, start, end, x=None):  # pragma: no cover
         start, end = f('', '2019-09-01T12:00Z', 'blah')
 
 
+def test_ensure_timestamps_omitted_args():
+    @utils.ensure_timestamps('start', 'end', 'other')
+    def f(start, end, a, *, other):  # pragma: no cover
+        return start, end
+
+    with pytest.raises(TypeError) as err:
+        start, end = f('2020-01-01T00:00Z')
+    assert str(err.value) == "missing a required argument: 'end'"
```

Comment:
I prefer using `raises(TypeError, match="a string")` but no objection

Reply:
huh, didn't know that was a thing